js/enable-select2.js

```diff
@@ -13,28 +15,43 @@ CRM.$(function() {
    * We're debouncing the callback to avoid calling the plugin multiple times
    * during DOM changes
    */
-  var observer = new MutationObserver(debounce(function() {
+  var observer = new MutationObserver(debounce(function () {
     CRM.$('select:visible:not(.no-select2):not(.crm-form-multiselect)')
-      .select2({
+    .each(function () {
```

Comment:
I think it would be better to add indentation here:

```js
CRM.$('select:visible:not(.no-select2):not(.crm-form-multiselect)')
>>.each(function () {
>>>>// ...
>>});
```